app/src/main/java/com/example/swugger/Event.java

```diff
@@ -1,52 +1,59 @@
 package com.example.swugger;
 
 import java.io.Serializable;
+import java.util.ArrayList;
 
 public class Event implements Serializable {
```

Comment:
Why not Parcable? It's faster becuase it doesn't need to do [reflection](https://www.geeksforgeeks.org/reflection-in-java/)

Reply:
implement Parcable? i'll look into it, serializable is just my goto cause of unity